pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -79,6 +79,11 @@ func (m *vmMutator) Create(_ *types.Request, newObj runtime.Object) (types.Patch
 		return nil, err
 	}
 
+	patchOps, err = m.ensureMaintenanceModeStrategyIsSet(vm, patchOps)
```

Comment:
as below comment, in `create` case, do not change user's input silently

Reply:
in the `create` case, the user's input will never be changed. The mutator will only add the label if it doesn't exist.